poutyne/framework/iterators.py

```diff
@@ -90,6 +93,7 @@ def __init__(self,
         self.initial_epoch = initial_epoch
         self.callback = callback
         self.metrics_names = metrics_names
+        self.batches_per_step = batches_per_step
```

Comment:
Do we really need to keep this value ?

Reply:
Mistake. Fixed by 874c3f709364a90f3c5b28faf6118b27fd3185f2